src/conn/conn_api.c

```diff
@@ -2266,37 +2268,79 @@ __wt_verbose_config(WT_SESSION_IMPL *session, const char *cfg[], bool reconfig)
+        verbose_value = (WT_VERBOSE_LEVEL)sval.val;
+        goto all_verbose_assign;
+    } else {
+        WT_RET_MSG(session, EINVAL, "Failed to parse verbose option '%s'", ft->name);
+    }
+
+all_verbose_assign:
+    for (ft = verbtypes; ft->name != NULL; ft++) {
+        conn->verbose[ft->flag] = verbose_value;
+    }
+
+    /* check if config other verbose, here shoud skip "all" */
     WT_RET(__wt_config_gets(session, cfg, "verbose", &cval));
     for (ft = verbtypes; ft->name != NULL; ft++) {
         ret = __wt_config_subgets(session, &cval, ft->name, &sval);
         WT_RET_NOTFOUND_OK(ret);
+        if (ft->flag == WT_VERB_ALL)
+            continue;
 
-        if (ret == WT_NOTFOUND)
+        if (ret == WT_NOTFOUND) {
             /*
              * If the given event isn't specified in configuration string, default it to the
              * WT_VERBOSE_NOTICE verbosity level. WT_VERBOSE_NOTICE being an always-on informational
              * verbosity message.
+             *
+             * If verbose have config the "config_all_verbose=xx", There's no need to default
+             * this config to the WT_VERBOSE_NOTICE verbosity level
+             *
+             * for examples:
+             *  1. If we config "verbose=[all:1, checkpoint:3]", the result is "all other config=1,
+             * checkpoint:3"
+             *  2. If we config "verbose=[checkpoint:3, all:1]", the result is alse "all other
+             * config=1, checkpoint:3"
              */
+            if (config_all_verbos_flag == true)
+                continue;
+
             conn->verbose[ft->flag] = WT_VERBOSE_NOTICE;
-        else if (sval.type == WT_CONFIG_ITEM_BOOL && sval.len == 0)
+        } else if (sval.type == WT_CONFIG_ITEM_BOOL && sval.len == 0) {
             /*
              * If no value is associated with the event (i.e passing verbose=[checkpoint]), default
              * the event to WT_VERBOSE_DEBUG_1. Correspondingly, all legacy uses of '__wt_verbose',
              * being messages without an explicit verbosity level, will default to
              * 'WT_VERBOSE_DEBUG_1'.
              */
             conn->verbose[ft->flag] = WT_VERBOSE_DEBUG_1;
-        else if (sval.type == WT_CONFIG_ITEM_NUM && sval.val >= WT_VERBOSE_INFO &&
-          sval.val <= WT_VERBOSE_DEBUG_5)
+        } else if (sval.type == WT_CONFIG_ITEM_NUM && sval.val >= WT_VERBOSE_INFO &&
+          sval.val <= WT_VERBOSE_DEBUG_5) {
             conn->verbose[ft->flag] = (WT_VERBOSE_LEVEL)sval.val;
-        else
+        } else {
             /*
              * We only support verbosity values in the form of positive numbers (representing
              * verbosity levels e.g. [checkpoint:1,rts:0]) and boolean expressions (e.g.
              * [checkpoint,rts]). Return error for all other unsupported verbosity values e.g
              * negative numbers and strings.
              */
             WT_RET_MSG(session, EINVAL, "Failed to parse verbose option '%s'", ft->name);
```

Comment:
I was thinking about a small improvement here:
```suggestion
            WT_RET_MSG(session, EINVAL,
              "Failed to parse verbose option '%s' with value '%" PRId64 "'", ft->name, sval.val);
```